src/scss/skin-modern/components/_settingspanel.scss

```diff
@@ -68,6 +68,56 @@
       display: none;
     }
   }
+
+  .#{$prefix}-ui-subtitlesettingtoggle {
+    @extend %ui-button;
+
+    background-image: svg('assets/skin-modern/images/settings.svg');
+    padding: .3em;
+
+    &:hover {
+      @include svg-icon-shadow;
+    }
+
+    &.#{$prefix}-on {
+      background-image: svg('assets/skin-modern/images/settingsX.svg');
+    }
+  }
+
+  .#{$prefix}-ui-subtitleoptioncloser {
```

Comment:
Rename to `.#{$prefix}-ui-subtitlesettingsclosebutton` and extract to `_subtitlesettingsclosebutton.scss`